### PR TITLE
Issue #187: add 'bash' as type for source blocks

### PR DESCRIPTION
**Pull request by rhoads-zach**
@inoxx03 @Ladicek mind taking  a look?

### CODE REVIEW

**Comment by Ladicek** [comment]
+1 I think it's correct and it's the right thing to do, but it won't fix the original problem, will it?

File: docs/topics/secured-mission-common-interaction.adoc [hunk 1]
```diff
@@ -10,7 +10,7 @@ The Java client is packaged as an executable jar and is located at target/sso-cl
 To run the client, use `java -jar target/sso-client.jar -app ...`.
 The full set of command arguments that you can specify can be seen by passing -help:
 +
-[source,options="nowrap",subs="attributes+"]
+[source,bash,options="nowrap",subs="attributes+"]
 ----
 [sstark sso]$ java -jar target/sso-client.jar --help
```
Comment (inoxx03) [inline docs/topics/secured-mission-common-interaction.adoc]: @rhoads-zach Maybe we should remove sstark's username from the snippet here. WDYT?

**Comment by rhoads-zach** [comment]
@Ladicek I built it locally and the language no longer appears when you hover your mouse over it box, so i believe it does address the initial issue.

**Comment by Ladicek** [comment]
Ah, OK, good!

File: docs/topics/secured-mission-common-interaction.adoc (hunk 1 above)
Comment (rhoads-zach) [inline docs/topics/secured-mission-common-interaction.adoc]: @inoxx03 fixed

**Comment by rhoads-zach** [comment]
merged: https://github.com/openshiftio/appdev-documentation/commit/4dff338726be15aaf82544f9abfe4dfc571f35ef

**Pull request closed without merging**